io/disk/Avago_storage_adapter/avago9361_vd.py

```diff
@@ -156,6 +163,8 @@ def test_online_offline(self):
                 self.set_online_offline(state)
                 time.sleep(10)
         self.vd_delete()
+        cmd = "%s /c%d set autorebuild=on" % (self.tool, self.controller)
+        self.check_pass(cmd, "Failed to set auto rebuild on")
```

Comment:
Guess this is not needed, as they are already handled in `setUp()` and `tearDown()`